arangod/Agency/Node.cpp

```diff
@@ -912,8 +1075,6 @@ std::pair<Node const&, bool> Node::hasAsNode(std::string const& url) const {
     return good_pair;
   } catch (...) {
     // do nothing, fail_pair second already false
```

Comment:
Why are these Logs removed?

Reply:
Because exceptions are used for control flow